client/components/chart/utils.js

```diff
@@ -573,14 +586,20 @@ export const drawBars = ( node, data, params ) => {
 		.attr( 'width', params.xGroupScale.bandwidth() )
 		.attr( 'height', d => params.height - params.yScale( d.value ) )
 		.attr( 'fill', d => getColor( d.key, params ) )
+		.attr( 'tabindex', '0' )
+		.attr(
+			'aria-label',
+			d => `${ getTooltipDate( params, d.date ) } ${ d.key } ${ formatCurrency( d.value ) }`
```

Comment:
I have the same problem as above, the key is extra info & can be omitted

Reply:
I see/hear that this is a problem with the previous period, since `getTooltipDate( params, d.date )` will always get the "current" date, not the date of the previous period's value. Is there a way to change that? So that if I'm comparing August 1-31 vs July 1-31, it can tell me `August 6th, 2018 $500` and then `July 6th, 2018 $450`?

Currently it would say `August 6 2018 Last Month (Aug 1 - 31, 2018) $500`, then `August 6 2018 Previous Period (Jul 1 - 31, 2018) $450`.